x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/utils/helpers.test.tsx

```diff
@@ -1797,21 +1797,16 @@ describe('Exception helpers', () => {
         {
           id: 'host.name',
         },
-        {
-          id: RESPONSE_ACTIONS_ALERT_AGENT_ID_FIELD.endpoint,
-          label: 'Agent status',
-          overrideField: 'agent.status',
-        },
-        {
-          id: RESPONSE_ACTIONS_ALERT_AGENT_ID_FIELD.sentinel_one,
-          overrideField: 'agent.status',
-          label: 'Agent status',
-        },
-        {
-          id: RESPONSE_ACTIONS_ALERT_AGENT_ID_FIELD.crowdstrike,
-          overrideField: 'agent.status',
-          label: 'Agent status',
-        },
+        // Fields used in support of Response Actions
+        ...Object.values(RESPONSE_ACTIONS_ALERT_AGENT_ID_FIELDS)
```

Comment:
Q: Is this order dependant? What would happen if there's device.id (crowdstrike field) first, and then agent.id ? Would former get overwritten by the latter?